index.js

```diff
@@ -6,6 +6,17 @@ const request = require('request');
 const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
+const piwikTracker = require('./lib/express-piwik-tracker.js');
+
+// track all API requests with piwik middleware
+if (config.tracking.disableTracking !== 'true') {
```

Comment:
Strongly suggest to use [`yn`](https://www.npmjs.com/package/yn) to parse the configuration variable, so an environment variable `"TRUE"` would also work and we still have a proper boolean variable here to check against.

So in the configuration file you would have

```json
c.tracking.disableTracking = yn(env.DISABLE_TRACKING) || false;
```

and here you can simply do `if(config.tracking.disableTracking) {....`